spec/01-unit/02-conf_loader_spec.lua

```diff
@@ -262,6 +262,13 @@ describe("Configuration loader", function()
       assert.is_nil(err)
       assert.is_table(conf)
     end)
+    it("errors when specifying a port in cassandra_contact_points", function()
+        local conf, err = conf_loader(nil, {
+            cassandra_contact_points = "addr1:9042,addr2"
+        })
+        assert.equal("bad Cassandra contact point 'addr1:9042': port must be specified in cassandra_port", err)
+        assert.is_nil(conf)
+    end)
```

Comment:
unless I'm mistaken, if you add another test with 
```lua
cassandra_contact_points = [[some/really\bad/host\name,addr2]]
```
if will fail, because you don't check on `endpoint` and `err`